pandemania/src/components/ChartWrapper/ChartWrapper.jsx

```diff
@@ -37,30 +45,28 @@ const initialState = {
 			},
 		},
 	},
-	selection: 'one_year',
+	selection: timelineEnum.ALL,
 };
 
-const timelineEnum = Object.freeze({
-	ONE_MONTH: 'one_month',
-	SIX_MONTH: 'six_months',
-	ONE_YEAR: 'one_year',
-	YTD: 'ytd',
-	ALL: 'all',
-});
-
 const AreaDateTimeCountryData = ({ countryName }) => {
 	const [chartData, setChartData] = useState(initialState);
 
+	useEffect(() => {
+		fetchData();
+	}, [countryName]);
+
 	const updateData = (timeline) => {
+		const dates = chartData.series[0].data;
```

Comment:
This operation seems unsafe. Are we sure that it won't cause an exception when no data is fetched?

Reply:
Done